src/store/index.js

```diff
@@ -61,8 +58,9 @@ const store = createStore({
                     return response;
                 })
         },
-        fetchMovies({commit}, page) {
-            axios.get(`/movies?page=${page}`)
+        fetchMovies({commit}, data) {
+            console.log(data)
+            axios.get(`/movies?page=${data.page}&genre=${data.genre ? data.genre : -1}`)
```

Comment:
Sto saljes data.genre -1 ? To neki defaultni? Ako jeste onda bi trebalo to izbaciti posto ako nema definisan genre u URL trebalo bi sve vratiti